Given 'no users exist' do
  User.destroy_all
end
